subt/zmq-subt-x2.json

```diff
@@ -120,8 +122,10 @@
 
               ["app.pose2d", "lora.pose2d"],
               ["lora.cmd", "app.cmd"],
-              ["app.artf_xyz", "lora.artf"],
+              ["reporter.artf_all", "lora.artf"],
+              ["lora.artf_xyz", "reporter.artf_xyz"],
               ["lora.raw", "rosmsg.broadcast"],
+              ["rosmsg.radio", "lora.radio"],
```

Comment:
This does not make sense.

Reply:
why? In virtual the "raw" packets contain identification of source, i.e. it is not 1:1 to `raw`, so I used alternative input `radio`

Reply:
because I expected the lora driver (in osgar/drivers) to be a low level driver for a physical device